Sources/Views/Pie/PieChartSlice.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PieChartSlice.swift
+//  Delirium
+//
+//  Created by Jelle Vandebeeck on 23/08/16.
+//  Copyright © 2016 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+/// The `PieChartSlice` is responsible for a slice that is displayed in the pie chart.
+@objc public protocol PieChartSlice {
```

Comment:
class is ook gewoon een swift ding. Je voegt eigenlijk gewoon toe dat diegene die het protocol implementeert een classes moet zoen en geen stuct of... dus als er geen objective-c meer zou zijn blijft dat werken. maar met @objc verwijs je naar objective-c. Ik vind dat dus niet nodig om objective-c daar te zetten. Had je mijn comment ook zo begrepen. Ik had het dus niet over objective-c maar over puur swift blijven.